src/game/tick.c

```diff
@@ -130,8 +130,9 @@ static void advance_day(void)
     if (game_time_advance_day()) {
         advance_month();
     }
-    if (game_time_day() == 0 || game_time_day() == 8) {
-        city_sentiment_update();
+    int update_sentiment_cooldown = game_time_day() == 0;
+    if (update_sentiment_cooldown || game_time_day() == 8) {
+        city_sentiment_update(update_sentiment_cooldown);
```

Comment:
```suggestion
    if (game_time_day() == 0 || game_time_day() == 8) {
        city_sentiment_update();
```

By having more granular value changes and more cooldown ticks you can still update sentiment twice a month.

Reply:
If using int probably yes